src/main/java/com/tobiasdiez/easybind/optional/ObservableOptionalValue.java

```diff
@@ -91,21 +93,25 @@ default EasyObservableValue<T> asOrdinary() {
 
     /**
      * Returns a new observable that holds the same value as this observable when the value is present and matches the given predicate,
-     * otherwise empty.
+     * otherwise it holds an empty optional.
      *
      * @param predicate the predicate to apply to a value, if present
      * @throws NullPointerException if the predicate is {@code null}
      */
-    OptionalBinding<T> filter(Predicate<? super T> predicate);
+    OptionalBinding<T> filterOpt(Predicate<? super T> predicate);
```

Comment:
I cannot find anything about a new `filter` method in JavaFX 19. Is this really necessary?

Reply:
I aimed for consistency here

Reply:
Consistency is sadly the thing that we loose here anyway (e.g. there is still an `orElse` method). So please revert this.

Reply:
The only name clash was about the optional bindings, including orElse

Reply:
I would prefer if only the methods that really clash with the new javafx methods are renamed.